evap/staff/tests/test_forms.py

```diff
@@ -678,6 +679,21 @@ def test_handle_multivaluedicts(self):
         self.assertEqual(Questionnaire.objects.filter(contributions=self.contribution2).count(), 2)
 
 
+class CourseCopyFormTests(TestCase):
+    # pylint: disable=no-self-use
```

Comment:
We've used `@staticmethod` in other places -- any specific reason to not use it here?